data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,596 @@
+	sound "flamethrower"
+
+
+outfit "Remass Injector"
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 2760000
+	thumbnail "outfit/unknown"
+	"mass" 3
+	"outfit space" -3
+	"engine capacity" -3
+	"afterburner thrust" 72
+	"afterburner energy" 3
+	"afterburner heat" -12
+	"afterburner fuel" 0.3
+	"magnozzle rings" -1
+	description "A powerful magnetohydrodynamic turbopump forces a torrent of water into a fusion torch's magnetic nozzle, dramatically increasing thrust at the cost of a significant increase in propellant consumption. The additional mass flow helps reduce the waste heat imparted on a ship by the torch drive."
+	description "The Remass Injector is a component that can be installed into an Avgi fusion torch to give it afterburning capabilities. It requires a magnozzle ring slot to be installed."
+
+
+outfit "Inductive Extractor"
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 2760000
+	thumbnail "outfit/unknown"
+	"mass" 12
+	"outfit space" -12
+	"engine capacity" -3
+	"energy generation" 1.5
+	"heat generation" 1.5
+	"thrust" -9
+	"thrusting energy" -30
+	"thrusting heat" 6
+	"magnozzle rings" -1
+	description "A reinforced superconducting loop and power conditioning unit allows a significant amount of power to be inductively extracted from a fusion torch's exhaust as it expands against the field produced by the magnetic nozzle. This removes some energy from the exhaust plume and reduces its produced thrust, but the liberated energy is great enough that this is the primary means by which high performance Avgi starships generate power."
+	description "The Inductive Extractor is a component that can be installed into an Avgi fusion torch to turn a portion of its thrust into energy. It requires a magnozzle ring slot to be installed."
+
+
+outfit "XY-180 Radial Thrusters"
+	plural "XY-180 Radial Thrusters"
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 221000
+	thumbnail "outfit/rcs small"
+	"mass" 18
+	"outfit space" -18
+	"engine capacity" -18
+	"turn" 750
+	"turning energy" 1.5
+	"turning heat" -3
+	"reverse thrust" 9
+	"reverse thrusting energy" 1.5
+	"reverse thrusting heat" -3
+	"steering flare sprite" "effect/avgi flare/rcs"
+		"frame rate" 30
+		"scale" 0.0625
+	"steering flare sound" "plasma small"
+	"reverse flare sprite" "effect/avgi flare/rcs"
+		"frame rate" 30
+		"scale" 0.125
+	"reverse flare sound" "plasma small"
+	description `Avgi radial thrusters are hybrid thermal-electric thrusters, designed to allow translation in any direction. By using reactor heat to expel superheated water vapor out of a set of nozzles, they can help a ship cool off, but require the ship to have excess heat in order to function.`
+
+
+outfit "XY-360 Radial Thrusters"
+	plural "XY-360 Radial Thrusters"
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 473000
+	thumbnail "outfit/rcs large"
+	"mass" 36
+	"outfit space" -36
+	"engine capacity" -36
+	"turn" 1710
+	"turning energy" 3
+	"turning heat" -6
```

Comment:
Fascinatingly, the heat use of these engines does not gracefully degrade in a mixed-engine environment. It's all or nothing. This is a code problem, not an Avgi problem, but it's worth noting.